templates/infinispan-persistent.json

```diff
@@ -10,28 +10,37 @@
       "iconClass": "icon-infinispan",
       "tags": "java,database,datagrid,jboss",
       "template.openshift.io/documentation-url": "http://infinispan.org/documentation/",
-      "template.openshift.io/long-description": "Infinispan is a high performance, scalable, key/value data grid solution.",
+      "template.openshift.io/long-description": "This template provides a standalone Infinispan server (a high performance, scalable, key/value data grid solution) with persistence enabled. In order to provide custom configuration, please update $ApplicationName-configuration ConfigMap and restart a Pod.",
       "template.openshift.io/provider-display-name": "Red Hat, Inc.",
       "template.openshift.io/support-url": "http://infinispan.org"
     },
+    "labels": {
+      "template": "infinispan-persistent"
+    },
     "name": "infinispan-persistent"
   },
   "objects": [
     {
       "kind": "ServiceAccount",
       "apiVersion": "v1",
       "metadata": {
+      "labels": {
+        "template": "infinispan-persistent"
+      },
         "name": "${APPLICATION_NAME}"
       }
     },
     {
       "kind": "RoleBinding",
       "apiVersion": "v1",
       "metadata": {
-        "name": "view-${APPLICATION_NAME}"
+        "labels": {
+          "template": "infinispan-persistent"
+        },
+        "name": "${APPLICATION_NAME}-view"
       },
       "userNames": [
-        "system:serviceaccount:myproject:${APPLICATION_NAME}"
+        "system:serviceaccount:${NAMESPACE}:${APPLICATION_NAME}"
```

Comment:
@bparees is there any better way to do it? I would love to get rid of this `${NAMESPACE}` and this is the only place where I need it....